data-source/json/test_routing_not_affected_by_answers_not_on_path.json

```diff
@@ -115,10 +110,11 @@
             }, {
                 "type": "Interstitial",
                 "id": "valid-skipped-interstitial",
-                "title": "This page should have been skipped!",
-                "content": [{
-                    "description": ""
-                }],
+                "content": {
+                    "contents": [{
+                        "description": ""
```

Comment:
Bad find/replace?. Context missing from v3.

Reply:
Hmm. Not sure what happened there. I'll look through and find any others, but if you spot any let me know.